examples/01_icetray/04_i3_module_in_native_icetray_example.py

```diff
@@ -109,4 +97,28 @@ def main() -> None:
 
 
 if __name__ == "__main__":
-    main()
+    if not has_icecube_package():
+        Logger(log_folder=None).error(
+            "This example requires IceTray to be installed, which doesn't "
+            "seem to be the case. Please install IceTray; run this example in "
+            "the GraphNeT Docker container which comes with IceTray "
+            "installed; or run an example script in one of the other folders:"
+            "\n * examples/02_data/"
+            "\n * examples/03_weights/"
+            "\n * examples/04_training/"
+            "\n * examples/05_pisa/"
+            "\nExiting."
+        )
```

Comment:
The error message is repeated in [03_i3_deployer_example.py](https://github.com/graphnet-team/graphnet/pull/436/files#diff-87b0fafe123e0fb5adb98b894fc028649cb5715babf389285461b8bc03af458b)

Is it possible to store these error messages in a dedicated place? such as "error_message_IceTray" ,"error_message_PISA", etc.